DEVELOPMENT.md

```diff
@@ -198,6 +198,34 @@ to explicitly enable it.
 ko apply -f test/config/sugar.yaml
 ```
 
+## Running a Single Rekt Test with e2e-debug.sh
+
+To run a single rekt test using the `e2e-debug.sh` script, follow these instructions:
+
+1. Navigate to the project root directory.
+
+2. Execute the following command in your terminal:
+
+    ```bash
+    ./hack/e2e-debug.sh <test_name> <test_dir>
+    ```
+
+    Replace `<test_name>` with the name of the rekt test you want to run, and `<test_dir>` with the directory containing the test file.
+
+    **Example:**
+
+    ```bash
+    ./hack/e2e-debug.sh TestMyRektScenario test/rekt/scenarios
+    ```
+
+    This will run the specified rekt test (`TestMyRektScenario` in this case) from the provided directory (`test/rekt/scenarios`).
+
+    **Note:** Ensure that you have the necessary dependencies and configurations set up before running the test.
```

Comment:
I really like this explanation, but maybe it is worth mentioning that test_name is actually a regex? Otherwise people may be confused when more than one test runs some of the time. The explanation for the flag we use under the hood is:
> Run only those tests, examples, and fuzz tests matching the regular expression. For tests, the regular expression is split by unbracketed slash (/) characters into a sequence of regular expressions, and each part of a test’s identifier must match the corresponding element in the sequence, if any. Note that possible parents of matches are run too, so that -run=X/Y matches and runs and reports the result of all tests matching X, even those without sub-tests matching Y, because it must run them to look for those sub-tests.


WDYT @professorabhay , is it better for this to be a simple explanation and not mention that, or to let users know that they can use a regex here if they want?

Reply:
For me who always likes to deep dive in tech, mentioning that point will be a good option. 
But from the perspective of a normal user, keeping it simple will help the user to easily understand it. 🙂

What do you suggest? 💭

Reply:
Maybe leave it as it is and then add in a small note saying that the value is technically a regex, but normally they will just want to use the name of the test they would like to run. I feel like this might be a nice middle ground.